src/_posts/security/procedures/2000-01-01-external-testing.md

```diff
@@ -38,8 +45,7 @@ For example:
 #### Procedure to follow
 
 1. Send the authorization document (signed by the customer and the pentesting
-   service provider) to the Security Team at the address security [at]
-   scalingo.com
+   service provider) to the Security Team at the address [security@scalingo.com](mailto:security@scalingo.com)
```

Comment:
is it safe to use mailto links? I thought this [at] technique was to prevent spam from web crawlers, but I'm too rusty on that and maybe times have changed and it no longer makes a difference :)

Reply:
Time have changed indeed. The current suggestion is to use a spam detection mechanism to prevent spam. The email address will eventually leak anyway. We already use email addresses everywhere on the documentation.